language-server/src/features/workspace.js

```diff
@@ -111,7 +113,7 @@ export default {
         ]
       });
     } else {
-      watchWorkspace(onWorkspaceChange);
+      watchWorkspace(onWorkspaceChange, isMatchedFile);
```

Comment:
The function you're passing here has a different signature than the one on line 67. I don't think this one will work. Also on line 125.

However, since the refactor, I don't think there's a reason to be passing a function to `watchWorkspace` or anywhere else we're passing that function. It probably needs to pass `schemaFilePatterns` instead.

Reply:
I checked and seems like it works even if do not pass this at all works too.

Reply:
Right, I'm passing schemaFilePatterns now

Reply:
Line 81 is still passing function.